### PR TITLE
chore: cleanup inactive owners

### PR DESCRIPTION
**Pull request by thesuperzapper**
This PR is an important cleanup of the OWNERS files.

It will reduce the project's exposure to potentially compromised GitHub accounts by removing access from users who no longer need it.

I have removed reviewers and approvers who have not interacted with the `kubeflow/kubeflow` repo in the last year, or who are no longer associated with the project due to moving companies.

To pay respect to historical approvers, I have moved them to the `emeritus_approvers` section, which does not convey any permission, but denotes that they once had approver access. (Read more about this in the [Kubernetes OWNERs docs](https://www.kubernetes.dev/docs/guide/owners/#emeritus)).

---

This PR will also reduce the frustration of people who make PRs because it will prevent inactive members from being assigned/tagged to review/approve them.

### CODE REVIEW

**Comment by thesuperzapper** [comment]
@kimwnasptd as you and I are the only active (semi) root approvers in this repo, I am interested in getting your approval before we merge.

**Review by SachinVarghese — APPROVED** [review]
/lgtm

**Comment by thesuperzapper** [comment]
@kimwnasptd btw, I am sure the tests failing are flakes, because I didn't make any code changes.

So once we agree, this should be safe to merge.

**Comment by kimwnasptd** [comment]
LGTM from my side, but would like to get an explicit ACK from the approvers that get moved that they indeed won't be able drastically change their contribution frequency in the next immediate months.

If we don't have an update in 2 weeks, let's merge this PR

**Comment by kimwnasptd** [comment]
cc @elikatsis @yanniszark @StefanoFioravanzo @tasos-ale @DavidSpek @discordianfish @PatrickXYS

**Comment by thesuperzapper** [comment]
@elikatsis thanks for your work in the past! If you ever want to participate again, you are more than welcome, even if it's just discussing ideas.

**Comment by yanniszark** [comment]
ACK from me as well! I'm following updates here and there but I don't have the time to be a maintainer right now.
Keep up the good work and tag me (or better message me in slack) if you need me :)

**Comment by StefanoFioravanzo** [comment]
Hey! Ack - I intend to get more involved as the new year begins but most probably not in a code contributor capacity. Thanks Matthew for taking care of this.

**Comment by thesuperzapper** [comment]
@kimwnasptd looks like we have had acks from everyone and it's also been 2 weeks.

Feel free to merge when ready.

**Comment by discordianfish** [comment]
Sorry, missed this. Yeah I probably won't be able to contribute much in the close future since currently I don't have any gigs that involve kubeflow. This might change though.

**Comment by thesuperzapper** [comment]
@kimwnasptd I think we are good to merge this now.

**Comment by thesuperzapper** [comment]
In the interest of security, and given the acceptance of affected parties I am going to move forward with merging this PR.

(Assuming the bot will let me)

/approve

**Comment by kimwnasptd** [comment]
A little bit late, but 
/lgtm

**Pull request merged**
